packages/extensions/src/shared/constants/extension-keywords.ts

```diff
@@ -0,0 +1 @@
+export const KEYWORD_EXTENSION = 'directus-extension';
```

Comment:
Are we going to make use of this in `extensions-sdk` & `extensions-registry`?

Reply:
Apparently this constant isn't used anywhere at all and there's two keywords, so I'll drop this one for now